extensions/blocks/membership-button/edit.jsx

```diff
@@ -84,10 +85,11 @@ class MembershipsButtonEdit extends Component {
 				const products = result.products;
 				const shouldUpgrade = result.should_upgrade_to_access_memberships;
 				const upgradeURL = result.upgrade_url;
+				const siteSlug = result.site_slug;
```

Comment:
This is starting to be a lot of `const` from that `result` object. Should we do something like this instead?

```js
const {
	connect_url: connectURL,
	products,
	should_upgrade_to_access_memberships: shouldUpgrade,
	upgrade_url: upgradeURL,
	site_slug: siteSlug,
	connected_account_id: connected
} = result;
```

Reply:
Whoa, I haven't seen renaming in deconstruct syntax! I have to try it!

Reply:
IMplemented